public/javascript/controllers/application.js

```diff
@@ -56,61 +33,47 @@ function bindClicks() {
 }
 
 function updateViews() {
-    updateDesktopVisbility();
-    viewer.updateViewer(activeMode, modes[activeMode]);
-    buttonUtil.markSelected('switchmode', activeMode);
 
-    if (activeMode === 'reader') {
-        buttonUtil.removeClassFromAttributeNameAndValue('showreaderactive', 'true', 'is-hidden');
-    } else {
-        buttonUtil.addClassToAttributeNameAndValue('showreaderactive', 'true', 'is-hidden');
-    }
-}
+    document.body.className = 'is-' + activeMode;
 
-function updateMode(newMode) {
-    var oldMode = activeMode;
-
-    if (newMode === 'desktop' && !desktopEnabled && window._previewEnv !== 'live') {
-        return;
+    if (desktopEnabled) {
+        document.body.className += ' desktop-enabled';
     }
 
+    viewer.updateViewer(activeMode, modes[activeMode]);
+    buttonUtil.markSelected('switchmode', activeMode);
+}
+
+function triggerAnalytics(oldMode, newMode) {
     if ((oldMode !== newMode) && modes[oldMode].isMobile && modes[newMode].isMobile) {
         analyticsCtrl.recordOrientationChange();
     }
 
-    if ((oldMode !== "desktop") && (newMode === "desktop")) {
+    if ((oldMode !== 'desktop') && (newMode === 'desktop')) {
```

Comment:
Perhaps we could enforce quote style in `.eslintrc` with `"quotes": [2, "single"]`?